v1/examples/NameClass/MultipleGivenMultipleFamily.json

```diff
@@ -0,0 +1,20 @@
+{
+    "nameParts": [
+    {
```

Comment:
Can I be really annoying and suggest we fix the indents here (we're missing a level for the list)? If you configure the editor with 2 space indents rather than 4, they shouldn't get out of hand ;-)